schemas/internal/init.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package littlehorse;
+
+option java_multiple_files = true;
+option java_package = "io.littlehorse.common.proto";
+
+import "google/protobuf/timestamp.proto";
+import "service.proto";
+import "acls.proto";
+
+// An object containing important server configurations that will be stored
+// in RocksDB and referenced after server restarts or rebalances.
+message InitConfig {
```

Comment:
Super pedantic comment. I think the `Config` part might be misleading. This isn't a configuration, but more of a log or "journal entry" of what happened.

Maybe we call it `InitializationHistory`?